src/types.ts

```diff
@@ -38,13 +54,25 @@ export enum Language {
 
 export interface GameStartType {
   slpVersion: string | null;
+  timerType: TimerType | null;
+  gameMode: GameModeType | null;
+  friendlyFireEnabled: boolean | null;
+  bombRainEnabled: boolean | null;
   isTeams: boolean | null;
-  isPAL: boolean | null;
+  itemSpawnBehavior: ItemSpawnBehaviorType | null;
+  selfDestructScoreValue: number | null;
   stageId: number | null;
+  startingTimerSeconds: number | null;
+  enabledItems: number | null;
+  damageRatio: number | null;
   players: PlayerType[];
-  scene: number | null;
-  gameMode: GameMode | null;
+  minorScene: number | null;
+  majorScene: number | null;
```

Comment:
```suggestion
  scene: number | null;
  majorScene: number | null;
```
Likewise over here, can we just keep minor scene as scene?